content/Foundation/Summit/2023/home.pml

```diff
@@ -31,8 +31,62 @@ Sept/Oct 2023, dates TBC
       Latest News
     </h3>
   </div>
+  <h3 class="highlight">
```

Comment:
I would suggest to change the bullet points above to have: 
- Latest News: Call for Presentations
- About
- [...]

or add the Call for Presentation as a new item in the bullet list

Reply:
I'll keep it in latest news for now, and give it its own item when we announce the venue

Reply:
👍